kong/tools/utils.lua

```diff
@@ -54,16 +54,37 @@ end
 
 local v4_uuid = uuid.generate_v4
 
+--- Generates a v4 uuid.
+-- @return string with uuid
+function _M.uuid() end -- just to trick Ldoc for documentation purposes
```

Comment:
no it doesn't, it's one of those Ldoc edge cases where it just doesn't behave very well.